scripts/scraper-ng/test/framework/utils.js

```diff
@@ -66,7 +66,57 @@ function recipePath(recipeName) {
   return path.join(recipesDir, recipeName + ".yaml");
 }
 
+/**
```

Comment:
If you're feeling ambitious, this could be done as a custom matcher for Jest in [jest.setup.js](https://github.com/mdn/stumptown-content/blob/master/scripts/scraper-ng/test/framework/jest.setup.js). The Jest matcher API is [a little bit complex](https://jestjs.io/docs/en/expect#custom-matchers-api) though, so it could happen in a separate PR

Reply:
I did think about this, but... the code to do this is quite a lot more complicated and it wasn't at all obvious to me that it was worth it.

Reply:
Fair enough 😃